src/audio/Kconfig

```diff
@@ -559,6 +559,25 @@ config COMP_MFCC
 	  control blob. Directory tools/tune/mfcc contains a tool to create
 	  the configurations.
 
+config DTS_CODEC
+	bool "DTS codec"
+	default n
+	select DTS_CODEC_STUB if COMP_STUBS
+	help
+	  Select to include DTS codec.
+	  In order to compile with this option enabled, a pre-compiled static library
+	  must be provided by DTS for your target platform. If this library is not present
+	  then compilation errors will occur.
+	  For more information, please contact sales@xperi.com
+
+config DTS_CODEC_STUB
+	bool "DTS codec stub"
```

Comment:
why does this option have to be user-selectable? The above `DTS_CODEC` will select this one if `COMP_STUBS` is enabled, and this option depends on `DTS_CODEC` anyway. And I assume it cannot be selected without `COMP_STUBS` either? (should it depend on it too?) So, this option - if at all needed - can be hidden?

Reply:
This would require @cujomalainey comment. 
And for this question,
"And I assume it cannot be selected without COMP_STUBS either?" The answer is yes. DTS_CODEC_STUB requires both DTS_CODEC and COMP_STUBS enabled.